src/main/java/core/basesyntax/strategy/CalculateStrategy.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class CalculateStrategy {
```

Comment:
Let's redesign this part of the architecture. 
Create an interface - Operation Handler, with only method - handle(FruitTransaction)
For each type of operation, implement this interface
In strategy class, you should keep correspondence table - which handler is responsible for which type of operation
In a for loop, when we are iterating thru the list of transactions, parserd by our Transaction  Parser, we, depending on the transaction type, use a specific handler (one of our implementations) (we get it from the look-up table/map by key) to handle it